src/en/controllers-backup.md

```diff
@@ -1,385 +1,278 @@
-best to simply back up the entire Juju directory (`~/.local/share/juju`).
-
-!!! Note: 
-    On Windows systems, the Juju directory is in a different place
-    (usually `C:\Users\{username}\AppData\Roaming\Juju`. Also, although `tar` is 
-    available for Windows, you may prefer to use a more Windows-centric backup 
-    application.
-
-### A simple backup of the Juju client directory
-
-As the Juju directory is simply just another directory on your filesystem, you 
-may wish to include it in any regular backup procedure you already have. For a 
-simple backup on an Ubuntu system, you can just use `tar` to create an archive 
-of the directory:
-
-```bash
-cd ~
-tar -cpzf juju-client-$(date "+%Y%m%d-%H%M%S").tar.gz .local/share/juju 
-```
-
-This command datestamps the created file for easy identification. You may, of
-course, call it what you wish. 
-
-!!! Note: 
-    As mentioned previously, the files in this backup include the keys 
-    Juju uses to connect to running environments. Anybody who has access to this 
-    backup will be able to connect to and use your controllers/models, so a 
-    further step of encrypting this backup file may be advisable.
- 
-### Restoring ~/.local/share/juju
-
-For Ubuntu, restoring your Juju client settings is a simple matter of
-extracting the archive you created above.
-
-```bash
-cd ~
-tar xzf juju-yymmdd-hhmmss.tar.gz 
-```
-
-!!! Note: 
-    This command will extract the contents of the archive and overwrite 
-    any existing files in the Juju directory. Please be sure that this is what you 
-    want!
-
+Title: Controller backups
+TODO:  Bug tracking: https://bugs.launchpad.net/juju/+bug/1771433
+       Bug tracking: https://bugs.launchpad.net/juju/+bug/1771426
+       Bug tracking: https://bugs.launchpad.net/juju/+bug/1771202
+       Bug tracking: https://bugs.launchpad.net/juju/+bug/1771673
+       Bug tracking: https://bugs.launchpad.net/juju/+bug/1771657
+       Bug tracking: https://bugs.launchpad.net/juju/+bug/1771674
+       Bug tracking: https://bugs.launchpad.net/juju/+bug/1771821
+       Put back HA section when HA feature set has stabalized
+
+# Controller backups
+
+Data backups can be made of the Juju client and the Juju controller. This page
+concerns itself with the controller aspect only. For information on client
+backups see the [Juju client][client-backups] page.
+
+A backup of a controller enables one to re-establish the configuration and
+state of a controller. It does **not** influence workload instances on the
+backing cloud. That is, if such an instance is terminated directly in the cloud
+then a controller restore cannot re-create it. However, as we'll see, a restore
+does have the ability to re-create a controller instance.
+
+This page will cover the following topics:
+
+ - Creating a backup
+ - Managing backups
+ - Restoring from a backup
+
+<!-- REMOVING THIS TEMPORARILY
+ - High availability considerations
+-->
 
 ## The Juju controller
```

Comment:
why do we still have this heading?

Reply:
It corresponds to the HA section that was temporarily removed.